js/src/whitebit.js

```diff
@@ -22,7 +22,7 @@ export default class whitebit extends Exchange {
             'name': 'WhiteBit',
             'version': 'v4',
```

Comment:
Can you please rever this file? It is automatically generated from the Ts version

Reply:
Reverted